examples/tutorial/auto_parallel/test_ci.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euxo pipefail
+
+pip install titans
+pip install pulp
+conda install coin-or-cbc
```

Comment:
Make it into a conda `environment.yaml` and include pytorch and colossalai.

Reply:
Ok